cozepy/model.py

```
@@ -402,6 +402,197 @@ async def build(
+
+    @abc.abstractmethod
+    def get_items(self) -> List[T]:
+        raise NotImplementedError
+
+    @property
+    @abc.abstractmethod
+    def response(self) -> HTTPResponse:
+        raise NotImplementedError
+
+
+class TokenPaged(PagedBase[T]):
```

Pylint (3.3.7)

[refactor] Too many instance attributes (8/7)

(R0902)